Fix/caesar cipher

wjhurley opened the pull request:
<!-- FreeCodeCamp Pull Request Template -->

<!-- IMPORTANT Please review https://github.com/FreeCodeCamp/FreeCodeCamp/blob/staging/CONTRIBUTING.md for detailed contributing guidelines -->

<!-- Help with PRs can be found at https://gitter.im/FreeCodeCamp/HelpContributors -->

<!-- Make sure that your PR is not a duplicate -->
#### Pre-Submission Checklist

<!-- Go over all points below, and after creating the PR, tick all the checkboxes that apply. -->

<!-- All points should be verified, otherwise, read the CONTRIBUTING guidelines from above-->

<!-- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [x ] Your pull request targets the `staging` branch of FreeCodeCamp.
- [x ] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x ] You have only one commit (if not, [squash](http://forum.freecodecamp.com/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [x ] All new and existing tests pass the command `npm run test-challenges`. Use `git commit --amend` to amend any fixes.
#### Type of Change

<!-- What type of change does your code introduce? After creating the PR, tick the checkboxes that apply. -->
- [x ] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)
#### Checklist:

<!-- Go over all points below, and after creating the PR, tick the checkboxes that apply. -->

<!-- If you're unsure about any of these, don't hesitate to ask in the Help Contributors room linked above. We're here to help! -->
- [ ] Tested changes locally.
- [x ] Closes currently open issue (replace XXXX with an issue no): Closes #10365
#### Description

<!-- Describe your changes in detail -->

Updated last test for Caesar Cipher from 'JUMPED' to 'JUMPS' per the issue discussion stating that the tests did not use all letters of the alphabet. Change made directly on GitHub interface because I have had challenges getting a local copy to work properly.

wjhurley committed:
Changed last test per issue discussion

wjhurley committed:
Revert "Changed last test per issue discussion"

This reverts commit e7b57b9896db29fb3258167099d2f071487d3f98.

dhcodes committed:
Improve check for disabled tag for jQuery challenge

wjhurley committed:
Update last test from 'JUMPED' to 'JUMPS'

Update last test and corresponding message for Caesar Cipher from 'JUMPED' to 'JUMPS' per the issue discussion.

wjhurley committed:
Update last test from 'JUMPED' to 'JUMPS'

Update last test and corresponding message for Caesar Cipher from 'JUMPED' to 'JUMPS' per the issue discussion.

ltegman commented:
Once your squash your commits this looks good to go 👍 

Just FYI, in the future you can avoid needing to squash your commits when you correct something by `--amend`ing your last commit instead of reverting it.

arialaurel committed:
Merge pull request #10412 from dhcodes/fix/improved-test-disabling-element-jQuery

Improve test on jQuery disable challenge

theshubhamp-zz committed:
Description of "Filter Arrays with .filter" updated

Fixes issue #10695
The description of the challenge "Filter Arrays with .filter" was updated to remove double negatives from the description as pointed out in the issue #10695

dhcodes committed:
Merge pull request #10730 from shubham1996/fix/rewrote-double-negative

Description of "Filter Arrays with .filter" updated

wjhurley committed:
Merge branch 'fix/caesar-cipher' of https://github.com/wjhurley/FreeCodeCamp into fix/caesar-cipher
